src/engine/renderer/tr_bsp.cpp

```diff
@@ -4007,6 +4007,20 @@ static void R_LoadFogs( lump_t *l, lump_t *brushesLump, lump_t *sidesLump )
 
 static void R_SetDefaultLightGrid()
 {
+	if ( glConfig2.max3DTextureSize == 0 )
```

Comment:
SetDefaultLightGrid is only called by LoadLightGrid, so this code is unreachable.